internal/provider/resource_namespace_file.go

```diff
@@ -0,0 +1,200 @@
+package provider
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func resourceNamespaceFile() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manages a Kestra Namespace File.",
+
+		CreateContext: resourceNamespaceFileCreate,
+		ReadContext:   resourceNamespaceFileRead,
+		DeleteContext: resourceNamespaceFileDelete,
+		Schema: map[string]*schema.Schema{
+			"tenant_id": {
+				Description: "The tenant id.",
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+			},
+			"namespace": {
+				Description: "The Namespace file namespace.",
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+			},
+			"filename": {
+				Description: "The path to the namespace file that will be created.\n" +
```

Comment:
```suggestion
				Description: "The local path to the namespace file that will be deployed.\n" +
```
to show it's a path on a local machine, not on kestra

I often find it confusing in some CLIs so worth clarifying :)

Reply:
oh wait, this is the target path on kestra VS Code? and the source is the local path?

if so, perhaps worth changing the naming to source_path and target_path, or local_path + kestra_path?